projects/packages/jetpack-mu-wpcom/src/features/wpcom-admin-interface/removed-calypso-screen-notice.tsx

```diff
@@ -67,7 +67,7 @@ const Notice = () => {
 						<>
 							<div
 								className="removed-calypso-screen-notice__image"
-								style={ { backgroundImage: `url(${ bgPattern })` } }
+								style={ { backgroundImage: `url(${ backgroundPattern })` } }
```

Comment:
Now that we have a static URL, I guess this can be part of the CSS styles now:

https://github.com/Automattic/jetpack/blob/8e463d3f1ce0bdaff4186ef0a6bfe873090af5b8/projects/packages/jetpack-mu-wpcom/src/features/wpcom-admin-interface/removed-calypso-screen-notice.scss#L4-L11

Reply:
Good suggestions. I updated the styling.